src/main/resources/db/changelog/changelog.sql

```diff
@@ -310,3 +310,35 @@ CREATE TABLE `AssessmentTags` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
 
 CREATE INDEX `AssessmentTags-TagValue` ON `AssessmentTags`(`tagValue`);
+
+-- changeset bridge:14
+
+CREATE TABLE `ExternalResources` (
+  `guid` varchar(255) NOT NULL,
+  `appId` varchar(255) NOT NULL,
+  `assessmentId` varchar(255),
```

Comment:
Can you create a resource without an assessment? Does a foreign key make sense here?

Reply:
Eventually I think you could, by adding another FK column... however you cannot enforce the FK constraint once there are different FK columns to different tables. For example, we could add a "protocolGuid" column if they also had external resource links. I'm not 100% sure we'll need this, but I didn't want to create something that makes it impossible later.